src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -49,71 +42,86 @@
     (range (rand-int 50))))
 
 
+
+
+
+(react/defc ReactImportSelectedMC
```

Comment:
This is an object, so should be named as such. I suggest `ModalImportOptionsAndButton` for now so it asks for refactoring.